ecommerce/enterprise_coupons/utils.py

```diff
@@ -0,0 +1,30 @@
+"""
+Helper methods for enterprise app.
+"""
+import logging
+
+from django.conf import settings
+
+log = logging.getLogger(__name__)
+
+
+def get_build_mfe_base_url(catalog_uuid, base_url):
+    """
+    Build the base URL for the ecommerce microfrontend receipt page with catalog parameter.
+
+    Args:
+        catalog_uuid (UUID): The catalog UUID to include in the microfrontend URL.
+        base_url (str): The base URL of the LMS instance.
+
+    Returns:
+        str: The complete URL for the ecommerce microfrontend receipt page with the catalog parameter appended.
+
+    Raises:
+        ValueError: If MFE_REDIRECT_URL is not configured in settings.
+    """
+    mfe_path = getattr(settings, "MFE_REDIRECT_URL", None)
```

Comment:
I think we should call this ENTERPRISE_COUPONS_MFE_URL, to make more explicit its contents and use. Also, each new Django setting should at least have a default definition in base.py. I think we can use the local URL, something in terms of localhost:2003 (not necessarily the actual port). @daviidco